.sti/bin/assemble

```diff
@@ -201,6 +237,8 @@ if [ -d $LOCAL_SOURCE_DIR/deployments ]; then
   cp $LOCAL_SOURCE_DIR/deployments/*.war $DEPLOY_DIR >& /dev/null
 fi
 
+install_jrebel $DEPLOY_DIR
```

Comment:
it would be nice to have a way how to disable JRebel as well, please consider adding an env var 'DISABLE_JREBEL' or something similar.